Exiled.API/Features/ExiledDamageHandler.cs

```diff
@@ -0,0 +1,66 @@
+// -----------------------------------------------------------------------
+// <copyright file="ExiledDamageHandler.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    using Footprinting;
+
+    using PlayerStatsSystem;
+
+    /// <summary>
+    /// A special <see cref="AttackerDamageHandler"/> that allows you to define the attacker.
+    /// </summary>
+    public class ExiledDamageHandler : AttackerDamageHandler
+    {
+        private Footprint attacker;
+        private float damage;
```

Comment:
Why do these exist? Can't both `Attacker` and `Damage` properties made as `{ get; set; }` like they are made in base-game?

Reply:
Because calling virtual methods (property setters are in fact methods) in a constructor is typically a bad idea. 

In this particular case there's no issue since we know ExiledDamageHandler is the last inherited type, and that the 'methods' won't access things that could be null/etc in the inherited type, so it would be safe, but 1. Better to do it the right way.
2. The compiler will just do this itself anyways
3. Better to be safe than sorry later on if something changes.

Reply:
I see